addons/account/models/account_move.py

```diff
@@ -2545,6 +2545,7 @@ def _post(self, soft=True):
 
         # Create the analytic lines in batch is faster as it leads to less cache invalidation.
         to_post.mapped('line_ids').create_analytic_lines()
+        # Need to post in order to compute name
```

Comment:
Not sure the comment is needed with your history rewrite

Reply:
Was on my radar to remove this. Left it in as a reminder for the review.
I tried to separate the `account.move` write on post - Because we mark `posted_before` here, then update the line label after - so it is considered a change (but shouldn't be)
Too many tests failed by splitting it into 2 writes.
Any thoughts on improving things here?

Reply:
I'm not sure it is really an issue. The ugly thing was more the `False`, which is not here anymore. Seems to be goo for TSB too or did he not test this version?

Reply:
it was tested, and it's fine. better than the `False`. but ideally it shouldn't log on creation - should be only for updates after it was posted.